main.tex

```diff
@@ -18,6 +18,49 @@
 \begin{document}
 
 \maketitle
+% 2.3 in der Gliederung
+\subsection{Start in ausgewählten Regionen}
 
+Im Anschluss an den formalen Start fingen erneute Debatten um "Net-Neutrality" an Beoden zu gewinnen. Facebooks Dienst sei nicht konform mit dem Bestreben eines freien Internets, trotz der Bewerbung des Produkts als "kostenloses Facebook"\cite[3]{prasad2017}
+ , später als ein "step towards digital equality".
+
+\medskip
+
+In der ländlichen Bevölkerung gab es auch geteilte Meinungen hinsichtlich des Zugangs zum Internet, zumindest bei jenen, die sich selbst informiert haben oder (von zum Beispiel Nachrichtendiensten wie "Digit.in") \textcite{digitYT}
```

Comment:
> (von zum Beispiel [...]

Ist zwar richtig, musste aber den Satz zwei mal lesen um das zu verstehen. Vlt wäre es übersichtlicher in der Klammer "z.B." anstatt "zum Beispiel" zu nehmen.